c/meterpreter/source/extensions/sniffer/sniffer.c

```diff
@@ -129,179 +189,190 @@ DWORD request_sniffer_interfaces(Remote *remote, Packet *packet)
 		*/
 	DWORD result = ERROR_SUCCESS;
 
-	HANDLE hCfg;
-	unsigned int idx = 1;
+	char errbuf[PCAP_ERRBUF_SIZE+4];
+	int aidx = htonl(1);				// :~(
+	struct ifaces_list *ifaces;

```

Comment:
I think this (and the other code referencing ifaces in this function) can be removed without changing the behavior of the function...